src/mediasoupclient.cpp

```diff
@@ -17,17 +14,14 @@ namespace mediasoupclient
 
 		MSC_DEBUG("mediasoupclient v%s", Version().c_str());
 
-		webrtc::field_trial::InitFieldTrialsFromString("WebRTC-SupportVP9SVC/EnabledByFlag_3SL3TL/");
```

Comment:
@jmillan I moved the field trials init from being global to being local - since the global API got deprecated in WebRTC. Not sure if this is the default nowadays, I left it in to ensure compatibility.

Reply:
Just to follow up on this: The flag is - as far as I can tell - not included in WebRTC M140, so this should serve no function anymore (most likely it is indeed enabled by default now)